server/src/main/scala/drt/server/feeds/lgw/ResponseToArrivals.scala

```diff
@@ -0,0 +1,96 @@
+package drt.server.feeds.lgw
+
+import java.io.ByteArrayInputStream
+import drt.shared.Arrival
+import org.apache.commons.io.IOUtils
+import org.slf4j.{Logger, LoggerFactory}
+import scala.util.{Failure, Success, Try}
+import scala.xml.Node
+
+case class ResponseToArrivals(data: Array[Byte], locationOption: Option[String] ) {
+  val log: Logger = LoggerFactory.getLogger(getClass)
+
+  def getArrivals: List[(Arrival, Option[String])] = {
+    val is = new ByteArrayInputStream(data)
+    val xmlTry = Try(scala.xml.XML.load(is)).recoverWith {
+      case e: Throwable => log.error(s"Cannot load Gatwick XML from the response ${data.toString}", e); null
```

Comment:
recover with null is probably a bad idea here, why not just throw the exception? Or return an option?